src/client/containers/PullRequestSummary/IssuesSection/index.js

```diff
@@ -33,9 +33,9 @@ type IssuesSectionProps = {
 
 export const getIssues = (state: Object, props: Object): Object =>
   createSelector(
-    getPullRequest,
-    pr => ({
-      issues: pr && pr.issues ? pr.issues.filter(x => x.status !== IssueStatus.OBSOLETE) : [],
```

Comment:
This selector fetches PullRequest object with issues as a list of ids (instead of a list of objects). I don't think it was intended (since we are filtering the objects by status attribute), hence the change.